components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -96,6 +96,10 @@ object typing {
     }
   }
 
+  case object TypedNull extends TypingResult {
```

Comment:
Maybe it could be helpful to have another trait `WithValue { def value: Any }` implemented by both `TypedNull` and `TypedObjectWithValue` and value will be just `null` here? WDYT?